tensorflow/python/kernel_tests/lookup_ops_test.py

```diff
@@ -3309,6 +3309,75 @@ def testMutableHashTableFindHighRank(self):
       result = self.evaluate(output)
       self.assertAllEqual([[0, 1], [-1, -1]], result)
 
+  def testMutableHashTableFindWithInvalidShapeDefaultValue(self):
+    with self.cached_session():
+      default_val = [-1, -1]
+      table = lookup_ops.MutableHashTable(dtypes.string, dtypes.int64,
+                                          default_val)
+
+      input_string = constant_op.constant([["brain", "salad"],
+                                           ["tank", "tarkus"]])
+
+      raised_error = ValueError
+      if context.executing_eagerly():
```

Comment:
You can employ something like https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/kernel_tests/array_ops/gather_op_test.py#L285 instead....

Reply:
OK～